src/js/shepherd.js

```diff
@@ -11,6 +11,9 @@ const {
 
 let Shepherd = new Evented;
 
+const UNDEFINED = 'undefined';
```

Comment:
If the optimization here is to not have to write ```'undefined'``` everywhere, then we're better off creating a helper ```isUndefined()``` method to this check for us. This will reduce redundancy even further.

```javascript
const isUndefined = (obj) => {
  return typeof obj === 'undefined';
};
```